### PR TITLE
Git Homework

### PR DESCRIPTION
**Pull request by ferdyantoch**
Add several folders and files related with Git Homework

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+assignments: assignments/**
+github: .github/**
+
+text: ['*.txt']
+python: ['*.py']
```
Comment (ferdyantochandra) [inline .github/labeler.yml:5]: please add a newline here
```suggestion
python: ['*.py']

```

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+    - uses: actions/labeler@v4
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (ferdyantochandra) [inline .github/workflows/labeler.yml:15]: please add a new line here
```suggestion
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
        
```

**Review by ferdyantochandra — CHANGES_REQUESTED** [review]
Hi @ferdyantoch, I have some comments for your PR. Kindly check!

**Comment by ferdyantoch** [comment]
Hi @ferdyantochandra, I've updated the files. Thanks for your suggestion!

**Review by ferdyantochandra — APPROVED** [review]
LGTM!

**Pull request merged**